### PR TITLE
Feature/d3 asim 2401

### PR DESCRIPTION
**Pull request by spyrostz**

### CODE REVIEW

File: d3a_api_client/aggregator.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        self.device_uuid_list = []
+        self.aggregator_uuid = None
+        user_aggrs = self.list_aggregators()
+        for a in user_aggrs:
+            if a["name"] == self.aggregator_name:
+                self.aggregator_uuid = a["uuid"]
+        if self.aggregator_uuid is None:
+            aggr = self._create_aggregator()
+            self.aggregator_uuid = aggr["uuid"]
+        self.start_websocket_connection()
```
Comment (hannesdiedrich) [inline d3a_api_client/aggregator.py:58]: Minor but i would like it this to be refactored into its own function called "connect_to_simulation" or something like this. But skip it if you dont agree.

File: tests/test_aggregator.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import logging
```
Comment (hannesdiedrich) [inline tests/test_aggregator.py:1]: Could you please leave a TODO here, that this needs to be use in an integrationtest? Thanks!

Comment (spyrostz) [reply tests/test_aggregator.py:1]: Sure, leaving it now

File: d3a_api_client/aggregator.py (hunk 1 above)
Comment (spyrostz) [reply d3a_api_client/aggregator.py:58]: Will do, thanks

**Pull request merged**